tnc/data_handler.py

```diff
@@ -1830,20 +1914,24 @@ def burst_watchdog(self):
         DATA BURST
         """
         # IRS SIDE
-        if not static.ARQ_STATE or static.ARQ_SESSION_STATE != 'connected' or static.TNC_STATE != 'BUSY' or not self.is_IRS:
+        # TODO: We need to redesign this part for cleaner state handling
+        # return only if not ARQ STATE and not ARQ SESSION STATE as they are different use cases
+        if not static.ARQ_STATE and static.ARQ_SESSION_STATE != 'connected' or not self.is_IRS:
```

Comment:
okay, I removed the nested if state check. Hopefully this isn't breaking arq session.

Reply:
It doesn't appear to, I just tested it on my VM-VM setup.